src/controllers/shared/common.resume.submission.controller.ts

```diff
@@ -51,7 +52,15 @@ export const getResumePage = async (req: Request, res: Response, next: NextFunct
             [config.PAYMENT_REQUIRED_HEADER]: config.PAYMENTS_API_URL + config.PAYMENTS
           }
         };
-        const redirectPath = await startPaymentsSession(req, session, transactionId, overseaEntityId, headersPaymentUrl);
+
+        let baseURL: string;
+        if (isRegistration){
+          baseURL = `${config.CHS_URL}${config.REGISTER_AN_OVERSEAS_ENTITY_URL}`;
+        } else {
+          baseURL = `${config.CHS_URL}${config.UPDATE_AN_OVERSEAS_ENTITY_URL}`;
+        }
```

Comment:
You can cut down the number of lines here by using a ternary operator 

```suggestion
     let baseURL = `${config.CHS_URL}${isRegistration ? config.REGISTER_AN_OVERSEAS_ENTITY_URL : config.UPDATE_AN_OVERSEAS_ENTITY_URL}`;

```

Reply:
wow, looks great, made the change!